### ISSUE TITLE
[scripting] app.range.editableImages nil Cel error on drawImage

### ISSUE THREAD

**Issue by not-surt** [aseprite/aseprite#3069]
When using drawImage on an Image object from app.range.editableImages or app.range.images the following error occurs:
> Using a nil 'Cel' object

Demo script (run with an image open):
```
-- works
-- local images = {app.activeCel.image}

-- error: "Using a nil 'Cel' object" on drawImage
local images = app.range.editableImages
-- local images = app.range.images

for _,image in ipairs(images) do
    assert(image ~= nil)
    local work = Image(image)
    image:drawImage(work)
end
print("DONE!")
```

Code works fine when given app.activeCel.image instead.

### Aseprite and System version

* Aseprite version: v1.3-beta7-x64, Steam
* System: Windows 10 Pro, Version 10.0.19043 Build 19043

**Comment by Gasparoken** [comment]
Thanks for this report. I was able to reproduce it. It's a weird bug, because sometimes it works, some time it doesn't.

**Comment by dacap** [comment]
Not only it might generated random results, it can crash Aseprite. Fixed for the next release v1.2.31.
